### ISSUE TITLE
[Grida Canvas] clipboard decode on windows fails with `<!--StartFragment-->`

### ISSUE THREAD

**Issue by softmarshmallow** [gridaco/grida#379]
<img width="1041" alt="Image" src="https://github.com/user-attachments/assets/45a5d389-1a41-41c6-a428-152ed0f6a0b6" />

(I don't own a PC)

on Windows machine, writing text/html payload adds dirty fragments [link](https://learn.microsoft.com/en-us/windows/win32/dataxchg/html-clipboard-format).

The `<!--StartFragment-->` and `<!--EndFragment-->` are messing up with the xml tree, causing failure of parse. to cater this, either

- add a pre-defined query map
- make query attribute based, not path based.
